types/apollo-upload-client/index.d.ts

```diff
@@ -1,22 +1,58 @@
-// Type definitions for apollo-upload-client 8.1
+// Type definitions for apollo-upload-client 14.1
 // Project: https://github.com/jaydenseric/apollo-upload-client#readme
-// Definitions by: Edward Sammut Alessi <https://github.com/Slessi>
+// Definitions by: Edward Sammut Alessi <https://github.com/Slessi>, tyankatsu <https://github.com/tyankatsu0105>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.1
+// TypeScript Version: 3.6
 
-import { ApolloLink } from "apollo-link";
-import { HttpOptions } from "apollo-link-http-common";
+// ==============================================================================
+// imports
+// ==============================================================================
 
-export { ReactNativeFile } from "extract-files";
+import { ApolloLink } from '@apollo/client/core';
+import { HttpOptions as OriginHttpOptions } from '@apollo/client/link/http';
+import { isExtractableFile as OriginIsExtractableFile } from 'extract-files';
+
+// ==============================================================================
+// declare
+// ==============================================================================
 
 declare global {
     interface GlobalFetch {
-        fetch: WindowOrWorkerGlobalScope["fetch"];
+        fetch: WindowOrWorkerGlobalScope['fetch'];
     }
 }
 
+// ==============================================================================
+// types
+// ==============================================================================
+
+export type HttpOptions = OriginHttpOptions &
+    Partial<{
+        isExtractableFile: typeof OriginIsExtractableFile;
+        formDataAppendFile: typeof formDataAppendFile;
+    }>;
+
+export interface FormDataFileAppender {
+    formData: FormData;
+    fieldName: string;
+    file: any;
```

Comment:
I don't approve of `any`. I am assuming the correct type would be `ExtractableFile` from `@types/extract-files`

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7e8c74275cd90806f1a7973648fc04196a29879d/types/extract-files/index.d.ts#L21

Reply:
I see.

Reply:
done